### PR TITLE
Add THROUGH write type to DataFlow doc

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/Architecture-DataFlow.md [hunk 1]
```diff
@@ -296,3 +296,9 @@ while still persisting the data.
 <p align="center">
 <img src="{{ '/img/dataflow-async-through.gif' | relativize_url }}" alt="ASYNC_THROUGH data flow"/>
 </p>
+
+### Write to UFS Only (`THROUGH`)
+
+With `THROUGH`, data is written to under storage synchronously without being cached to Alluxio
+workers. This write type ensures that data will be persisted after write completes, but the speed
```
Comment (ZacBlanco) [inline docs/en/Architecture-DataFlow.md]: minor grammatical fix: " data will be persisted after **the** write completes,..."

Comment (apc999) [reply docs/en/Architecture-DataFlow.md]: done

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please

**Pull request merged**